### PR TITLE
Adding "zip" as a valid gTLD

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-24463

## Description
As `zip` is now a valid gTLD we need to not filter it out.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by bakatzir** [comment]
Dror and I talked about it and approved this change going forward. we will monitor the feedback from the field.

File: Packs/CommonScripts/Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def get_fqdn(the_input):
     fixed = get_tld(the_input, fail_silently=True, as_object=True, fix_protocol=True)
     domain = fixed or get_tld(the_input, fail_silently=True, as_object=True)
 
-    if domain and domain.tld != 'zip':
+    if domain:
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py:59]: let's mention in a comment the removal and why.

Comment (Ni-Knight) [reply Packs/CommonScripts/Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py]: Done

**Review by DeanArbel — APPROVED** [review]
🙈

**Pull request merged**